src/mobility/components/VehicleCardStat.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import {StyleSheet} from '@atb/theme';
+import {View} from 'react-native';
+import {ThemeIcon} from '@atb/components/theme-icon';
+import {ThemeText} from '@atb/components/text';
+import {SvgProps} from 'react-native-svg';
+
+type Props = {
+  icon: (SvgProps: SvgProps) => JSX.Element;
+  stat: string;
+  description: string;
+};
+
+export const VehicleCardStat = ({stat, description, icon}: Props) => {
```

Comment:
We might have already a component with an icon, title and subtitle...

Reply:
i dont think we have actualy